includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -381,8 +381,6 @@ public function display_pay_for_order_page_html( $order ) {
 		];
 
 		wp_localize_script( 'WCPAY_PAYMENT_REQUEST', 'wcpayPaymentRequestPayForOrderParams', $data );
-
-		$this->display_payment_request_button_html();
```

Comment:
For future eyes: This is removed from here because it injects a div in the page with the same id expected for Apple/Google Pay creating the button in the top of the page instead of close to WooPay and other express payments.